doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -174,7 +174,7 @@ It's time to deploy our own Econia Faucet to the local chain:
 git clone https://github.com/econia-labs/econia.git # only if necessary
 cd ./econia/src/move/faucet
 aptos init --profile econia_faucet_deploy # enter "local" for the chain
-export FAUCET_ADDR=<ACCOUNT-FROM-ABOVE>
+export FAUCET_ADDR=<ADDR-FROM-ABOVE> # make sure to put 0x at the start
```

Comment:
The reason it's ACCOUNT and not ADDR is because someone got confused and put the node (or faucet, forget which) _web_ address. I'd leave it as ACCOUNT.

Reply:
@elliottdehn addressed in 51e108c